plugins.xml

```diff
@@ -12669,4 +12669,142 @@ the registration of DOIs with mEDRA.</p>]]></description>
 			<description locale="pt_BR">Garante que a pergunta esteja relacionada ao revista atual.</description>
 		</release>
 	</plugin>
+	<plugin category="oaiMetadataFormats" product="biblat">
+		<name locale="es">Formato BIBLAT de Metadatos</nam
+		<name locale="en">BIBLAT Metadata Format</name>
+		<homepage>https://biblat.unam.mx/en/</homepage>
+
+		<summary locale="es">Provides a metadata structure consistent with the BIBLAT metadata format.</summary>
+		<summary locale="en">Estructura los metadatos de forma que son consistentes con el formato BIBLAT.</summary>
+
+		<description locale="es"><![CDATA[<p>Este modulo aggrega un formato de los metadatos para Bibliografía Latinoamericana.</p>]]></description>
+		<description locale="en"><![CDATA[<p>This module adds a metadata format for Bibliografía Latinoamericana.</p>]]></description>
+
+		<maintainer>
+			<name>Bibliografía Latinoamericana</name>
+			<institution>UNAM</institution>
+			<email>biblat@dgb.unam.mx</email>
+		</maintainer>
+
+		<release date="2024-01-01" version="2.3" md5="49e2c0aa6b4032e0bf8e19ec9b3519a9">
```

Comment:
La Galería solo funciona con los productos de versión 3.x, no 2.x, así que no necesitamos esta versión.
The plugin gallery only supports 3.x, not 2.x, releases, so we don't need this version to be listed.

Reply:
Quite versiones para 2.x